sentry-kotlin-multiplatform/src/androidUnitTest/kotlin/io/sentry/kotlin/multiplatform/BaseSentryTest.kt

```diff
@@ -0,0 +1,15 @@
+package io.sentry.kotlin.multiplatform
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+actual abstract class BaseSentryTest {
+    actual val platform: String = "Android"
+    actual val authToken: String? = System.getenv("SENTRY_AUTH_TOKEN")
```

Comment:
IIRC `System.getenv` returns empty if no env. var. found, maybe we need to check that, or the usage of `authToken` should check empty/null.
Right now it does only `assertNotNull(authToken)` but empty is valid.